bakerydemo/settings/base.py

```diff
@@ -191,3 +437,91 @@
+# URLs. Please set to the domain that users will access the admin site.
+if "PRIMARY_HOST" in os.environ:
+    WAGTAILADMIN_BASE_URL = "https://{}".format(os.environ["PRIMARY_HOST"])
+
+
+# Basic authentication settings
+# These are settings to configure the third-party library:
+# https://gitlab.com/tmkn/django-basic-auth-ip-whitelist
+if os.environ.get("BASIC_AUTH_ENABLED", "false").lower().strip() == "true":
+    # Insert basic auth as a first middleware to be checked first, before
+    # anything else.
+    MIDDLEWARE.insert(0, "baipw.middleware.BasicAuthIPWhitelistMiddleware")
+
+    # This is the credentials users will have to use to access the site.
+    BASIC_AUTH_LOGIN = os.environ.get("BASIC_AUTH_LOGIN", "wagtail")
+    BASIC_AUTH_PASSWORD = os.environ.get("BASIC_AUTH_PASSWORD", "wagtail")
+
+    # Wagtail requires Authorization header to be present for the previews
+    BASIC_AUTH_DISABLE_CONSUMING_AUTHORIZATION_HEADER = True
+
+    # This is the list of hosts that website can be accessed without basic auth
+    # check.
+    if "BASIC_AUTH_WHITELISTED_HTTP_HOSTS" in os.environ:
+        BASIC_AUTH_WHITELISTED_HTTP_HOSTS = os.environ[
+            "BASIC_AUTH_WHITELISTED_HTTP_HOSTS"
+        ].split(",")
+
+    BASIC_AUTH_RESPONSE_TEMPLATE = "base/basic_auth.html"
+
+
+# Logging
+# This logging is configured to be used with Sentry and console logs. Console
+# logs are widely used by platforms offering Docker deployments, e.g. Heroku.
+# We use Sentry to only send error logs so we're notified about errors that are
+# not Python exceptions.
+# We do not use default mail or file handlers because they are of no use for
+# us.
+# https://docs.djangoproject.com/en/stable/topics/logging/
+LOGGING = {
+    "version": 1,
+    "disable_existing_loggers": False,
+    "handlers": {
+        # Send logs with at least INFO level to the console.
+        "console": {
+            "level": "INFO",
+            "class": "logging.StreamHandler",
+            "formatter": "verbose",
+        },
+    },
+    "formatters": {
+        "verbose": {
+            "format": "[%(asctime)s][%(process)d][%(levelname)s][%(name)s] %(message)s"
+        }
+    },
+    "loggers": {
+        "bakerydemo": {
+            "handlers": ["console"],
+            "level": "INFO",
+            "propagate": False,
+        },
+        "wagtail": {
+            "handlers": ["console"],
+            "level": "INFO",
+            "propagate": False,
+        },
+        "django.request": {
+            "handlers": ["console"],
+            "level": "WARNING",
+            "propagate": False,
+        },
+        "django.security": {
+            "handlers": ["console"],
+            "level": "WARNING",
+            "propagate": False,
+        },
+    },
+}
+
+SHOW_DEMO_BANNER = os.getenv("SHOW_DEMO_BANNER", "false").lower() == "true"
+
+# Changing this? Be sure to update the initial data fixture
+DEFAULT_ADMIN_PASSWORD = "changeme"
```

Comment:
Question: Now we're preventing people from changing the default admin password - should we use a different password which encourages them _to_ change it?

Reply:
`admin`?

Reply:
`password`?
basically anything that is not `changeme`

Reply:
I've changed to `admin`, simply because it's the same as the username.